### PR TITLE
Feature/tri 573 irs api monitoring

### PR DESCRIPTION
**Pull request by ds-ext-ooyeneye**

### CODE REVIEW

File: chart/grafana/values.yaml [hunk 1]
```diff
@@ -147,3 +147,4 @@ dashboardProviders:
 dashboards:
   minio: dashboards/minio-dashboard.json
   outbound: dashboards/irs-outbound-requests.json
+  irsmonitoring: resource-monitoring-dashboard.json
```
Comment (ds-jkreutzfeld) [inline chart/grafana/values.yaml:150]: Where are the two new dashboards?

**Review by ds-jkreutzfeld — CHANGES_REQUESTED** [review]

**Comment by ds-jkreutzfeld** [comment]
What is the difference between those panels?
image

**Comment by ds-jkreutzfeld** [comment]
And why are there 6 jobs created, but the number of jobs in the Jobstore is 0? Minio has been running for two weeks, so there should be no job reset.

**Comment by ds-jkreutzfeld** [comment]
Looking at these panels, I have no idea what to read from that.
image
Can you explain?

**Comment by ds-jkreutzfeld** [comment]
What does this panel mean?
image

**Comment by ds-jkreutzfeld** [comment]
Which methods are described here?
image

File: chart/grafana/values.yaml (hunk 1 above)
Comment (ds-ext-ooyeneye) [reply chart/grafana/values.yaml:150]: Very strange, they are there in the commit project
image

**Comment by ds-ext-ooyeneye** [comment]
> 

JobStore works find in test, I will need to find out why the JobStore is 0

**Comment by ds-ext-ooyeneye** [comment]
> And why are there 6 jobs created, but the number of jobs in the Jobstore is 0? Minio has been running for two weeks, so there should be no job reset.

Duplicate one will be remove

**Comment by ds-ext-ooyeneye** [comment]
A new dashboard is available for review

**Comment by ds-ext-ooyeneye** [comment]
> Looking at these panels, I have no idea what to read from that. image Can you explain?

The new dashboard can be reviewed

**Comment by ds-ext-ooyeneye** [comment]
> And why are there 6 jobs created, but the number of jobs in the Jobstore is 0? Minio has been running for two weeks, so there should be no job reset.

The created are counter while job in JobStore is gauge, the counter continue to increase the gauge show the actual content available

**Comment by ds-ext-ooyeneye** [comment]
> Looking at these panels, I have no idea what to read from that. image Can you explain?

Had been remove, a bit confusing

**Comment by ds-ext-kmassalski** [comment]
When Job state changes from Running to Completed, running counter is not being decreased.

**Pull request closed without merging**